package.json

```diff
@@ -10,9 +10,9 @@
     "postinstall": "patch-package"
   },
   "devDependencies": {
-    "@sentry-internal/eslint-config-sdk": "9.10.1",
-    "@sentry-internal/eslint-plugin-sdk": "9.10.1",
-    "@sentry-internal/typescript": "9.10.1",
+    "@sentry-internal/eslint-config-sdk": "10.4.0",
```

Comment:
l (feel free to ignore): We released 10.5.0 in the meantime. Just FYI, leaving it up to you if you wanna bump again or not. No critical changes this time 😅